docs/users/cli-usage.md

```diff
@@ -117,13 +117,24 @@ Your project can be reached at: http://my-drupal8-site.ddev.local
 
 To get started using ddev with a TYPO3 project, clone the project's repository and checkout its directory.
 
+**Git Clone Example**
+
 ```
 git clone https://github.com/example-user/example-typo3-site
 cd example-typo3-site
 ```
 
 If necessary, run build steps that you may require, like `composer install` in the correct directory.
 
+**Composer Setup Example**
+
+```
+composer create-project typo3/cms-base-distribution my-typo3-site ^9
```

Comment:
The [official docs](https://docs.typo3.org/typo3cms/InstallationGuide/QuickInstall/Composer/Index.html) would have us use Stable 8.7, rather than 9. What do you think?

Reply:
You are right, we should use ^8 until v9 is released in October.